Source/ThirdParty/Diligent/Common/CMakeLists.txt

```diff
@@ -63,7 +67,7 @@ target_include_directories(Diligent-Common
 PRIVATE
     include
 PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/interface>
+    interface
```

Comment:
I am a bit surprised this does not cause issues. I thought CMake would complain about local path being in includes when SDK is being created. But i suppose you would have noticed that it is the case, so if nothing screams this must be ok? But idk how/why it would work.

Reply:
I have extracted this patch into our CMakeFiles.txt, it still works exactly like before.
I just don't want to patch 20 of Diligent CMakeFiles